### PR TITLE
Add instructions to rename a country

### PR DESCRIPTION
**Pull request by binaryberry**
Based on notes from @steventux

### CODE REVIEW

File: source/manual/rename-a-country.html.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+We recommend testing out these instructions in integration and then staging before using them in production.
+
+### 1. Update Foreign Travel Advice publisher
+
+This will update www.gov.uk/foreign-travel-advice/countryname. In [Travel Advice Publisher](https://github.com/alphagov/travel-advice-publisher):
+
+1. Create a pull request with:
+
+* A migration to update the relevant `country_slug` of TravelAdviceEdition. [Example] (https://github.com/alphagov/travel-advice-publisher/pull/346/commits/b28ff7b4eae96543324f61be700dca32f1ffdba5)
```
Comment (steventux) [inline source/manual/rename-a-country.html.md]: The `Example` link doesn't render as expected, is there whitespace between `[Example]` and `(https://github.com/alphagov/travel-advice-publisher/pull/346/commits/b28ff7b4eae96543324f61be700dca32f1ffdba5)` ?

File: source/manual/rename-a-country.html.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* A migration to update the relevant `country_slug` of TravelAdviceEdition. [Example](https://github.com/alphagov/travel-advice-publisher/pull/346/commits/b28ff7b4eae96543324f61be700dca32f1ffdba5)
+* A change of the relevant name and slug in the `lib/data/countries.yml` file. Keep the same `content_id` and `email_signup_content_id`, and ensure the alphabetical order of the list is respected. [Example](https://github.com/alphagov/travel-advice-publisher/pull/346/commits/3eb10a8519638850760698992dd1f6467b041ab0)
+
+2. Deploy Travel Advice publisher
+
+Once the above PRs are ready, deploy Travel Advice Publisher. This should be an `app:migrate_and_hard_restart` deploy as a hard restart is required to update the yml file.
+
+3. Run rake tasks
+
+* Run `bundle exec rake publishing_api:republish_edition[country_slug]` to update the PublishingApi.
```
Comment (barrucadu) [inline source/manual/rename-a-country.html.md]: Is that the literal string "country_slug" or the slug?

File: source/manual/rename-a-country.html.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Once the above PRs are ready, deploy Travel Advice Publisher. This should be an `app:migrate_and_hard_restart` deploy as a hard restart is required to update the yml file.
+
+3. Run rake tasks
+* Run `bundle exec rake publishing_api:republish_edition[country_slug]` to update the PublishingApi.
+* Run `publishing_api:republish_email_signups:editions` to update email subscriptions.
+
+4. Update the search title
+In the [UI](https://travel-advice-publisher.publishing.service.gov.uk/admin), go to the country, create a new edition, tick "minor update" and update the Search title and Search description fields to ensure search results have the updated name. Save and publish.
+
+### 2. Update Content Tagger
```
Comment (cbaines) [inline source/manual/rename-a-country.html.md]: I'd consider calling this Worldwide, or Worldwide taxons, as it's not Content Tagger that needs updating, it's just the content that it's ended up managing.

File: source/manual/rename-a-country.html.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Create a pull request with:
+* A migration to update the relevant `country_slug` of TravelAdviceEdition. [Example](https://github.com/alphagov/travel-advice-publisher/pull/346/commits/b28ff7b4eae96543324f61be700dca32f1ffdba5)
+* A change of the relevant name and slug in the `lib/data/countries.yml` file. Keep the same `content_id` and `email_signup_content_id`, and ensure the alphabetical order of the list is respected. [Example](https://github.com/alphagov/travel-advice-publisher/pull/346/commits/3eb10a8519638850760698992dd1f6467b041ab0)
+
+2. Deploy Travel Advice publisher
+Once the above PRs are ready, deploy Travel Advice Publisher. This should be an `app:migrate_and_hard_restart` deploy as a hard restart is required to update the yml file.
+
+3. Run rake tasks
+* Run `bundle exec rake publishing_api:republish_edition[country_slug]` to update the PublishingApi.
```
Comment (thomasleese) [inline source/manual/rename-a-country.html.md]: You could have links here to the job in jenkins, for example:

https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION=publishing-api&MACHINE=publishing-api-1.backend&RAKE_TASK=publishing_api:republish_edition[country_slug]

File: source/manual/rename-a-country.html.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+* Run `publishing_api:republish_email_signups:editions` to update email subscriptions.
```
Comment (thomasleese) [inline source/manual/rename-a-country.html.md]: This could also be a link to the rake task.

File: source/manual/rename-a-country.html.md (hunk 2 above)
Comment (binaryberry) [reply source/manual/rename-a-country.html.md]: It's the new slug! I've changed it to new_country_slug, which will hopefully make it clearer it needs to be substituted.

File: source/manual/rename-a-country.html.md (hunk 3 above)
Comment (binaryberry) [reply source/manual/rename-a-country.html.md]: True! Amended now.

**Pull request merged**